backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+  updatedAt                    DateTime          @updatedAt @map("updated_at") @db.Timestamp(6)
+  placeName                    String?           @map("place_name")
+  city                         String?
+  county                       String?
+  state                        String?
+  street                       String?
+  street2                      String?
+  zipCode                      String?           @map("zip_code")
+  latitude                     Decimal?          @db.Decimal
+  longitude                    Decimal?          @db.Decimal
+  alternateContact             AlternateContact?
+  applicantWorkAddress         Applicant?        @relation("applicant_work_address")
+  applicantAddress             Applicant?        @relation("applicant_address")
+  applicationsAlternateAddress Applications?     @relation("applications_alternate_address")
+  applicationsMailingAddress   Applications?     @relation("applications_mailing_address")
+  householdMemberAddress       HouseholdMember?  @relation("household_member_address")
+  householdMemberWorkAddress   HouseholdMember?  @relation("household_member_work_address")
+  applicationDropOffAddress    listings[]        @relation("application_drop_off_address")
+  applicationMailingAddress    listings[]        @relation("application_mailing_address")
+  leasingAgentAddress          listings[]        @relation("leasing_agent_address")
+  applicationPickUpAddress     listings[]        @relation("application_pick_up_address")
+  buildingAddress              listings[]        @relation("building_address")
+
+  @@map("address")
+}
+
+// Note: [type, phone_number] formerly max length 16; [other_type, first_name, last_name] formerly max length 64; [agency] formerly max length 128
+model AlternateContact {
+  id               String        @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt        DateTime      @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt        DateTime      @updatedAt @map("updated_at") @db.Timestamp(6)
+  type             String?
+  otherType        String?       @map("other_type")
+  firstName        String?       @map("first_name")
+  lastName         String?       @map("last_name")
+  agency           String?
+  phoneNumber      String?       @map("phone_number")
+  emailAddress     String?       @map("email_address")
+  mailingAddressId String?       @unique() @map("mailing_address_id") @db.Uuid
+  address          Address?      @relation(fields: [mailingAddressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  applications     Applications?
+
+  @@map("alternate_contact")
+}
+
+model AmiChart {
+  id                 String               @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt          DateTime             @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt          DateTime             @updatedAt @map("updated_at") @db.Timestamp(6)
+  items              Json
```

Comment:
this was previously typed as `AmiChartItem` is that lost here? i see we have the relationship below

Reply:
good question, and this is now captured in a note

` @Column("jsonb")` is the db's column type, the type protection will have to be done during the create/update endpoint steps